PZ/WebClient/Connector.py

```diff
@@ -0,0 +1,140 @@
+import requests
+import json
+
+# from PZ.WebClient.models import *
+
+
+class Connector(object):
+    def __init__(self, url_adr):
+        self.url_adr = url_adr
+        self.response = object()
+        self.json_data = object()
+        self.payload = {}
+
+    def set_url(self, url_adr):
+        self.url_adr = url_adr
+
+    def set_request_args(self, payload):
+        self.payload = payload
+
+    def get_resources(self):
+        resources_list = []
+        resource = None
+        page = None
+        self.response = requests.get(self.url_adr, params=self.payload)
+        self.json_data = json.loads(self.response.text)
+
+        # for item in self.json_data['resources']:
```

Comment:
I understand that you comment it for sake of using this code later, but consider extracting this to methods instead of commenting